R/link_metadata.R

```diff
@@ -0,0 +1,29 @@
+#' Linking metadata of original dataset when adding datasets to the qPackage
```

Comment:
This function is likely not going to be necessary once the new data_evolution() function is added to qData. We maybe should delete the script or, if needed, we comment it out for now... @EstherPeev could you check with Bernhard please and see what we can do about this?